.circleci/config.yml

```diff
@@ -149,6 +149,10 @@ jobs:
           name: Run e2e navigation test
           command: |
             python -m scripts.run_e2e_tests --prod_env --deparallelize_terser --skip-install --suite="navigation"
+      - run:
+          name: Run e2e wipeout test
```

Comment:
Any specific reason for running this in this container?

Note: This container blocks other e2e test containers so adding more here will increase the delay in running child builds.

Reply:
Hmm, that's actually a bit confusing. Can we rename all of the e2e test suite names to make that a bit clearer? This seems like an easy thing to miss when changing these configurations.

Reply:
Done, moved, let's do the rename in a different PR.